Dockerfile

```diff
@@ -2,6 +2,12 @@ FROM public.ecr.aws/bitnami/golang
 
 RUN apt update; apt install -y make git curl bash jq awscli
 
+COPY ./go.env /custom_go.env
+
+RUN touch $(go env GOROOT)/go.env &&  \
+    cat /custom_go.env >> $(go env GOROOT)/go.env &&  \
+    rm /custom_go.env
+
```

Comment:
Why not change the base image to offical golang image ? That should already have this by default